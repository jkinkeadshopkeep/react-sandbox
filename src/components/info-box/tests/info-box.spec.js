import React from 'react';
import { render } from '@testing-library/react';
import InfoBox from "../index";

describe('Info Box Component', () => {
  it('should render info box component', () => {
    const { container } = render(<InfoBox/>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
