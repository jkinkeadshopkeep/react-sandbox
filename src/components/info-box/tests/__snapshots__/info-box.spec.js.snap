// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Box Component should render with standard class name 1`] = `
<div
  class="info-box"
>
  <div>
    <h1>
      React sandbox
    </h1>
    <p>
      React sandbox for personal training of with react related tasks
    </p>
  </div>
  <div>
    <div>
      078 0192841
    </div>
    <div>
      reactsandbox@test.com
    </div>
    <div>
      992 Testing React Ave, TestWood, Testy McTestville
    </div>
  </div>
</div>
`;
